### ISSUE TITLE
There are some ES6 syntaxes in the "dist/dom7.module.js" file, such as `const` or `let`.

### ISSUE THREAD

**Issue by BiosSun** [nolimits4web/Dom7#21]

**Comment by ccchapman** [comment]
This causes my script execution to halt in IE 11.

https://github.com/nolimits4web/dom7/blob/28b0db30d77fa765ccae9497a2f0e3dfbc2cf55f/dist/dom7.module.js#L16

**Comment by Hideer** [comment]
@BiosSun  I also encountered this problem, I reported an error in compression

**Comment by slorber** [comment]
agree, this is painful for me. was quite complicated to track down why my Gatsby site is not running on IE11/Edge, due to using "swiper", which depends on this lib

**Comment by Riccardo-Andreatta** [comment]
I have the same issue in my project, is there a quick solution about that?
